.github/workflows/main.yml

```
@@ -30,7 +31,7 @@ jobs:
           restore-keys: ${{ runner.os }}-composer-
 
       - name: Install dependencies
-        run: composer install
+        run: composer update
```

This must no change. Instead add a new step before this one that removes `composer.lock`, but only for PHP 8.0. Otherwise our dependencies are ever changing and we our tests becomes unpredictable (as already seen in the error reported).

```suggestion
        run: composer install
```

---

A better way to check if this projects work with his declared dependencies would be to have this king of tests strategy https://github.com/Ocramius/DoctrineBatchUtils/blob/2.2.x/.github/workflows/phpunit.yml

Are you agree with this ?

---

If you refer to "lowest", "highest", "locked", then I don't want them for this project. I agree that it can have some values for some lib. But not here, because we virtually have a single dependency that really matters. The rest of dependencies is only used for developers and there is no point to test all those variations for a development environnement.

tl;dr; it's overkill for this lib